.travis.yml

```diff
@@ -14,6 +14,11 @@ git:
 
 install:
     - pip install sphinx
+    - wget https://github.com/autotest/autotest/archive/0.16.3.tar.gz
```

Comment:
Can we set some fixed url for latest autotest?

    eg: https://github.com/autotest/autotest/archive/autotest-latest.tar.gz?